<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetGSLibCustomColor</class>
 <widget class="QWidget" name="WidgetGSLibCustomColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblColorSample">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinR">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinG">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinB">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnColorDialog">
     <property name="toolTip">
      <string>Choose color dialog...</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WidgetGSLibCustomColor</receiver>
   <slot>updateColorSample(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WidgetGSLibCustomColor</receiver>
   <slot>updateColorSample(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WidgetGSLibCustomColor</receiver>
   <slot>updateColorSample(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnColorDialog</sender>
   <signal>clicked()</signal>
   <receiver>WidgetGSLibCustomColor</receiver>
   <slot>openColorChooser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateColorSample(int)</slot>
  <slot>openColorChooser()</slot>
 </slots>
</ui>
